src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue

```diff
@@ -102,6 +103,10 @@ const state = reactive({
         EDIT: i18n.t('DASHBOARDS.CUSTOMIZE.VARIABLES.SUB_TITLE_EDIT'),
     })),
     selectedVariable: '' as string,
+    variableNams: computed<string[]>(() => {
```

Comment:
suggestion: fix typo please :)

```suggestion
    variableNames: computed<string[]>(() => {
```

Reply:
OMG... Thanks!!